api/libproject/devstart.rb

```diff
@@ -3235,10 +3235,10 @@ def run_genomic_extraction(cmd_name, *args)
     'The dataset to record in the DB as associated with this extraction (arbitrary but must exist).')
 
   op.add_typed_option(
-    '--person_ids ["id1, id2, id3"]',
+    '--person_id_file [person_ids.txt]',
```

Comment:
what's the format of this file, is it txt or csv? because in the logic below you skip the header row so just want to confirm.

Reply:
the text file I was given was derived from a CSV, and formatted like this:
```
sample_id
100000
100002
100003
```

If you think it would be clearer to accept a file with no header, I can make that change.

Reply:
that's ok, maybe we just need to document this somewhere, in a comment or a document.